examples/opf/tools/sp_plotter.py

```diff
@@ -330,17 +330,16 @@ def testSPNew():
   
   
   # Setup a SP
-  sp = FDRCSpatial2.FDRCSpatial2(
```

Comment:
This `sp_plotter` is expecting a path of `~/Desktop/ExperimentResults` to exist. I don't think this will ever work out of the box.

@david-ragazzi I think if you create the `~/Desktop/ExperimentResults` directory, you might be able to run the examples.